dll/win32/browseui/internettoolbar.cpp

```diff
@@ -715,9 +731,21 @@ HRESULT CInternetToolbar::LockUnlockToolbars(bool locked)
     int                                     bandCount;
     CDockSite                               *dockSite;
     HRESULT                                 hResult;
+    HKEY                                    hKey;
 
     if (locked != fLocked)
     {
+        if(RegCreateKeyEx(HKEY_CURRENT_USER, 
```

Comment:
Please change (in this PR):
1. Space between `if` and opening parenthesis;
2. Use explicit unicode functions (-W and `L"strings"`);
3. indentation.

Reply:
Thank you for reviewing my pull request so quickly! I have added a new commit implementing your suggestions in accordance with our [coding style](https://reactos.org/wiki/Coding_Style).  If you want me to make any more changes, please let me know!